draft-ietf-quic-transport.md

```diff
@@ -1738,6 +1738,14 @@ network, an endpoint verifies the following when an ACK_ECN frame is received:
 
 * The increase in ECT(0) and ECT(1) counters MUST be no greater than the number
   of packets newly acknowledged that were sent with the corresponding codepoint.
+  
+* If the increase in the ECT(0), ECT(1) and CE counters are greater than the 
+  number of newly ACKed packets, and received ACK's smallest acknowledged or 
```

Comment:
I would just say "ACK's smallest acknowledged" and drop "or gap indicated packet number"

Reply:
Implemented.